builders/deploy.yaml

```diff
@@ -152,3 +152,16 @@ steps:
     - --allow-unauthenticated
     - --service-account=download-metrics@${PROJECT_ID}.iam.gserviceaccount.com
     - --set-env-vars=PROJECT_ID=${PROJECT_ID},KEY_SERVER_URL=${_KEY_SERVER_URL}
+- name: 'gcr.io/cloud-builders/gcloud'
+  waitFor: [ '-' ]
+    args:
+      - functions
+      - deploy
+      - UploadKeysToEFGS
```

Comment:
Please Use naming convention that all starts with `Efgs`, so `EfgsUploadKeys`. I would also consider `EfgsUploadBatch`.

Reply:
I think that `EfgsUploadKeys` will be better. I want to upload keys and the batching is something specific for EFGS (in one function call can be send many of batches). Also in database are saved keys, not batches of keys.